extensions/amp-analytics/0.1/vendors.js

```diff
@@ -107,6 +107,44 @@ export const ANALYTICS_CONFIG = {
       'xhrpost': false,
       'image': true
     }
+  },
+
+  'chartbeat': {
+    'vars': {
+      'decayTime': 15
+    },
+    'requests': {
+      'host': 'https://ping.chartbeat.net',
+      'interval': '${host}/ping?h=${domain}&' +
+        'p=${canonicalPath}&' +
+        'u=${clientId}&' +
+        'd=${canonicalHost}&' +
+        'g=${uid}&' +
+        'g0=${sections}&' +
```

Comment:
> If this information is available in JSON-LD, we could read it from there. Is that something you guys would be interested in?

Ultimately, yes that would be something we're interested in. However, at this point we are trying to add as little complexity for the publisher as possible. In our existing data collection functionality, these are values set specifically for the publisher's Chartbeat implementation. If we move the configuration away from the Chartbeat amp-analytics tag, there will likely be confusion with implementation for publishers with limited dev resources.